_config.yml

```diff
@@ -134,17 +134,42 @@ keep_files:
 
 # Plug-ins
 plugins:
+  - jekyll-archives
+  - jekyll-feed
+  - jekyll-sitemap
   - jekyll-email-protect
   - jekyll-github-metadata
-  - jekyll-feed
   - jekyll-paginate-v2
   - jekyll/scholar
   - jekyll-twitter-plugin
   - jemoji
 
+# Sitemap settings
+defaults:
+  -
+    scope:
+      path:            "assets/**/*.*"
+    values:
+      sitemap:         false
+
 # Extras
 github: [metadata]
 
+# -----------------------------------------------------------------------------
+# Jekyll Archives
+# -----------------------------------------------------------------------------
+
+jekyll-archives:
+  enabled: [year, tags, categories]
```

Comment:
could you add a brief comment on how to enable/disable categories? if you users don't want to have any categories, is it sufficient to simply leave `enabled:` blank?